tracer/test/Datadog.Trace.Debugger.IntegrationTests/ProbesTests.cs

```diff
@@ -311,15 +311,28 @@ async Task RunPhase(ProbeDefinition[] snapshotProbes, ProbeAttributeBase[] probe
 
                 if (metricProbes.Any())
```

Comment:
This method is getting a bit too long - maybe extract out `VerifyMetricProbeResults`, ``VerifyLogProbeResults` and `VerifySpanProbeResults` into separate methods?

Reply:
Fixed in bf60f7c0eea54e2717bb2ca2e8d9b17861e12203